homeassistant/components/canary/sensor.py

```diff
@@ -177,6 +190,6 @@ def extra_state_attributes(self):
             elif reading <= 1.0:
```

Comment:
I'd just do it this way, then `None` is not needed.
```suggestion
            else:
```